environmental-footprint/environmental-footprint/concepts.mdx

```diff
@@ -0,0 +1,84 @@
+---
+meta:
+  title: Environmental Footprint - Concepts
+  description: Understand the fundamental concepts of the Environmental Footprint calculator at Scaleway.
+content:
+  h1: Concepts
+  paragraph: Understand the fundamental concepts of the Environmental Footprint calculator at Scaleway.
+tags: environmental-footprint impact environment datacentre
+dates:
+  validation: 2024-09-11
+---
+
+## DDV_DC (Datacenter Lifespan)
+
+This value indicates the lifespan of a datacenter, if the value is not available it can be estimated. For example `DDV_DC` = 25 years.
```

Comment:
```suggestion
This value indicates the lifespan of a datacenter; if the value is not available, it can be estimated. For example `DDV_DC` = 25 years.
```